Header.jsx

```diff
@@ -0,0 +1,42 @@
+import React from "react";
+import { View, Text, StyleSheet, Button } from "react-native";
+
+export const Headers = () => (
+  <View style={styles.container}>
+    <Text style={styles.left}>Cancel</Text>
+    <Text style={styles.main}> New task </Text>
+    <Text style={styles.right}>Save</Text>
+  </View>
+);
+
+const styles = StyleSheet.create({
+  container: {
+    height: 70,
+    paddingTop: 30,
+    backgroundColor: "rgb(31, 134, 255)",
+    display: "flex",
+    justifyContent: "space-between",
+    flexDirection: "row",
+  },
+  left: {
+    display: "flex",
+    textAlign: "left",
+    color: "#ffffff",
```

Comment:
Nice job! `color` defaults to `black`, so you did right on specifying this `white` color. You can also use `white` as a value, but I encourage you to always use **hex** as I mentioned before.

Install this extension in VS Code, it will make your life that much easier! 
https://marketplace.visualstudio.com/items?itemName=naumovs.color-highlight